cmd/synthetic-monitoring-agent/main.go

```diff
@@ -103,6 +105,7 @@ func run(args []string, stdout io.Writer) error {
 	flags.BoolVar(&config.AutoMemLimit, "enable-auto-memlimit", config.AutoMemLimit, "automatically set GOMEMLIMIT")
 	flags.BoolVar(&config.DisableK6, "disable-k6", config.DisableK6, "disables running k6 checks on this probe")
 	flags.Float64Var(&config.MemLimitRatio, "memlimit-ratio", config.MemLimitRatio, "fraction of available memory to use")
+	flags.StringVar(&config.FeatureString, "features", featuresPlaceholder, "deprecated: optional feature flags")
```

Comment:
Parsing it as a string now, makes setting a default value to detect a little easier.